api/client/service/update.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/swarm"
 	"github.com/docker/go-connections/nat"
+	shlex "github.com/flynn-archive/go-shlex"
```

Comment:
Correct me if I'm wrong, but I believe `import "github.com/flynn-archive/go-shlex"` allows the use of `shlex` to refer to the package.